grpc/jwt/jwt.go

```diff
@@ -134,11 +140,8 @@ func (ts *jwtAccessTokenSource) Token() (*oauth2.Token, error) {
 	_ = token.Set(jwt.IssuedAtKey, iat)
 	_ = token.Set(jwt.ExpirationKey, exp)
 	_ = token.Set(jwt.JwtIDKey, uuid.New().String())
-	h := jws.NewHeaders()
```

Comment:
Not sure why this was here. I tested how the final token looked before and now and they are the same

Reply:
Maybe the previous version didn't explicitly set the `kid` in header which is mandatory for us to locate the key for verification.